### PR TITLE
Add basic multi-level list support to Text.rakumod

### PR DESCRIPTION
**Pull request by hopkinsza**

### CODE REVIEW

**Comment by vrurg** [comment]
@hopkinsza Would it be possible for you to provide more information and take care of the failing tests?

**Comment by hopkinsza** [comment]
Apologies for using you as my guinea pig as I learn the github process -- this patch improves the output of Pod::To::Text by adding indentation for pod6 items based on their level.

For example, given the following script:

```
use Pod::To::Text;

=begin pod
=item item
=item2 item2
=item3 item3
=item item
=item2 item2
=end pod

say pod2text($=pod);
```

The current output would be:

```
  * item

  * item2

  * item3

  * item

  * item2
```

After this modification, the output is:

```
  * item

    * item2

      * item3

  * item

    * item2
```

File: lib/Pod/To/Text.rakumod [hunk 1]
```diff
@@ -18,7 +18,7 @@ sub pod2text($pod) is export {
         when Pod::Block::Para       { twrap( $pod.contents.map({pod2text($_)}).join("") ) }
         when Pod::Block::Table      { table2text($pod)               }
         when Pod::Block::Declarator { declarator2text($pod)          }
-        when Pod::Item              { item2text($pod).indent(2)      }
+        when Pod::Item              { item2text($pod).               }
```
Comment (JJ) [inline lib/Pod/To/Text.rakumod:21]: The `.` is a syntax error, that's what's making the errors pop up.

**Review by JJ — CHANGES_REQUESTED** [review]
Please mind the errors.

**Comment by hopkinsza** [comment]
Must open a new pull request since I foolishly deleted and remade my fork while troubleshooting -- and GitHub will not let me edit this one to use that new repo 😅. Sorry again for this! However, the error should be fixed.

@vrurg
See #4202.

**Pull request closed without merging**